### ISSUE TITLE
[Question] Is there a way to remove service response times/update data from public dashboard?

### ISSUE THREAD

**Issue by fools12** [statping-ng/statping-ng#304]
Is there a way to remove the section of the public dashboard that contains the response times/update data about services? [see image]

I'm monitoring a lot of services and having so many of these is becoming a hassle.

![image](https://github.com/user-attachments/assets/3090c86d-0d6b-450d-94fb-034d4e58ba3e)

**Comment by jemand771** [comment]
I don't think so. you _could_ try hiding these with custom css, but that's not ideal/pretty
